### ISSUE TITLE
Wrong insertion point 

### ISSUE THREAD

**Issue by jeffmaury** [gtache/intellij-lsp#117]
I have the following file:

```
# Configuration file
# key = value
quarkus.application.name=jeff
```

Please note that it has only 3 lines and the third one has no CR or LF
At the end of the third line, I press ENTER to got the a new fourth line
Once on the beginning of the first line, I ask for code assist and select one of the proposed choice.
The choosen text is added at the end of the third line.

I have found this is related to the conversion between LSP positions and editor offset.
A PR is about to come

**Comment by borkdude** [comment]
@jeffmaury Is my issue maybe related to this issue? #118

**Comment by gtache** [comment]
Should be fixed since 1.6.0
